### PR TITLE
Bug 1197750 - [Contacts] After updating contacts, the old one and new…

### PR DESCRIPTION
**Pull request by mancas**
… one both exist, but they are same.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b3f3671d86a3fccd8bb15a2d63675bb5e72d97c1

File: apps/communications/contacts/js/views/list.js [hunk 1]
```diff
@@ -391,6 +391,14 @@ contacts.List = (function() {
 
     var config = utils.cookie.load();
     if (config) {
+      if (config.shouldEvict) {
+        resetDom(function() {
+          Cache.evict(false /* undo applied cache */,
```
Comment (borjasalguero) [inline apps/communications/contacts/js/views/list.js]: Could you put this in several lines? Thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fc6724065fc8ab20169c7a9d17065a4de6ae0e0b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=51a8344093e0391748530b1fe182a77dad423beb

**Pull request merged**